apps/frontend/src/app/components/candidatePicker/index.ts

```diff
@@ -0,0 +1,3 @@
+import { CandidatePicker } from "./CandidatePicker";
+
+export default CandidatePicker;
```

Comment:
This is for faster navigation of the code. If there are several components in the directory you can quickly find the parent, without looking through each component.

When the project becomes a bit more mature, creating a barrel file for the components directory would also be nice: 
https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

Reply:
Ok i see now, looks better this way. But then we need to be persistant of the use of components/containers/views, so that everything does not become a component :)